### PR TITLE
Add two projects: TiDB for go and TiKV for Rust

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Mattermost](https://github.com/mattermost/mattermost-server/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22Up+For+Grabs%22+label%3A%22Difficulty%2F1%3AEasy%22+label%3A%22Tech%2FGo%22+) _(label: Up For Grabs, Difficulty/1:Easy, Tech/Go)_ <br> Open source Slack-alternative in Golang and React
 - [Docker/CLI](https://github.com/docker/cli/labels/exp%2Fbeginner) _(label: exp/beginner)_ <br> The Docker CLI
 - [Kubernetes](https://github.com/kubernetes/kubernetes/labels/good%20first%20issue) _(label: good first issue)_ <br> Production-Grade Container Scheduling and Management System
+- [TiDB](https://github.com/pingcap/tidb/labels/for%20new%20contributors) _(label: for new contributors, Difficulty: Introductory)_ <br> A distributed scalable Hybrid Transactional and Analytical Processing (HTAP) database
```
Comment (vvscode) [inline README.md]: Doesn't look like correct link

File: README.md [hunk 2]
```diff
@@ -213,6 +214,8 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Rust-Clippy](https://github.com/rust-lang-nursery/rust-clippy/labels/good%20first%20issue) _(label: good first issue)_  <br> A bunch of lints to catch common mistakes and improve Rust code
 - [Rustfmt](https://github.com/rust-lang-nursery/rustfmt/labels/good-first-issue) _(label: good-first-issue)_ <br> A tool for formatting Rust code according to style guidelines.
 - [Iron](https://github.com/iron/iron/labels/easy) _(label: easy)_ <br> An extensible, concurrent web framework for Rust
+- [TiKV](https://github.com/tikv/tikv/issues?q=is%3Aissue+is%3Aopen+label%3A%22D%3A+Easy%22) _(label: for new contributors, Difficulty: Easy. Ideal for beginners, with mentoring available!)_ <br> A distributed transactional key-value database
```
Comment (vvscode) [inline README.md]: can you please use the same format for links as around? directly to labeled issues, not filter

File: README.md (hunk 1 above)
Comment (queenypingcap) [reply README.md]: Thanks! @vvscode ! Could you please elaborate on what's wrong with this link? It seems working from my side. I changed the description to make it consistent with the others.

File: README.md (hunk 2 above)
Comment (queenypingcap) [reply README.md]: Changed. Please take a look. Thanks!

**Comment by MunGell** [comment]
Thanks!

**Pull request merged**